aws_compliance.py

```diff
@@ -222,7 +222,7 @@ def reboots_required():
     failReason = ""
     offenders = []
     control = "reboots_required"
-    description = "Instances requiring a reboot, see /var/log/pay-reboots-required.log on instances"
+    description = "Instances requiring a reboot, see /var/log/pay-reboots-required.log on instances. See guidance https://pay-team-manual.cloudapps.digital/manual/suppor"
```

Comment:
Did half this url get chopped off somewhere?

Reply:
Yes 😬